CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 6.0.3 (2022-08-08)
```

Comment:
minor nitpick, but this'll likely get merged in today, so the date should be `2022-08-09`

Reply:
True! I'll fix the version and Changelog 👍